hyp3_gamma/insar/ifm_sentinel.py

```diff
@@ -367,17 +375,25 @@ def insar_sentinel_gamma(reference_file, secondary_file, rlooks=20, alooks=4, in
     log.info("Starting phase unwrapping and geocoding")
     unwrapping_geocoding(reference, secondary, step="man", rlooks=rlooks, alooks=alooks)
 
+    if water_masking:
+        mask = None
+        tiffiles = glob.glob("./*.tif")
+        for tiffile in tiffiles:
+            if mask is None:
+                mask = get_water_mask(tiffile, reference_file, mask_value=1)
+            apply_water_mask(tiffile, reference_file, mask=mask)
```

Comment:
```suggestion
    if water_masking:
        tiffiles = glob.glob("./*.tif")
        mask = get_water_mask(tiffiles[0], reference_file, mask_value=1)
        for tiffile in tiffiles:
            apply_water_mask(tiffile, reference_file, mask=mask)
```

sicne your only getting the mask based on the first tif

Reply:
changed. smart way!